Junk4Dollars-app/API/ApiServices.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+public class ApiServices {
```

Comment:
After thinking more about it, I do think that these would be better off split, however we can defer on that.